sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js

```diff
@@ -2986,7 +2986,7 @@ var setParentClass = function(child, parent) {
             var size = reference.rows * reference.cols;
             for (var i = 0; i < reference.rows; i++)
                 for (var j = 0; j < reference.cols; j++)
-                    ret.set(i, j, output[size * index + j * reference.cols + i]);
+                    ret.set(i, j, output[size * index + j * reference.rows + i]);
```

Comment:
Welcome back!

I don't think it's apparent why ret.set(i, j, output[size * index + j * reference.rows + i]) is logically correct whereas ret.set[i, j, output[size * index + i * reference.cols + j]) is not.  It really depends on how the data is laid out in |output|.  Can you explain a bit more and maybe add a comment in the code here?